spec/models/reviewable_spec.rb

```diff
@@ -106,6 +106,15 @@
         user.update_columns(moderator: false, admin: true)
         expect(Reviewable.list_for(user, status: :pending)).to eq([reviewable])
       end
+
+      it 'Let us filter by the target_created_by_id attribute' do
```

Comment:
This test only shows that the filter will exclude. We should also have a test that returns the particular item for that id.

Reply:
I changed this test with the one that you suggest

Reply:
Now you have the opposite problem. The filter might not do anything! :)

I was trying to suggest you test two things. The new test below + a test that passes a weird id and then you check for nothing.

Reply:
I added a second test to check  for exclusion :+1: